apps/cookbook/src/app/showcase/calendar-showcase/calendar-showcase.component.html

```diff
@@ -44,4 +44,46 @@ <h4>Properties:</h4>
 
   <h4>Events:</h4>
   <cookbook-api-description-events [events]="events"></cookbook-api-description-events>
+
+  <h4>Locales:</h4>
+
+  <p>
+    As mentioned in the <code>properties</code> table it's possible to provide additional existing
+    locales or custom ones. But before we take a look at how to provide them, let's take a look at
+    how locales are specified. The selected locale should be injected into the
+    <code>kirby-calendar</code> component. It can be done in the <code>AppModule</code> like this:
+  </p>
+
+  <cookbook-code-viewer [ts]="moduleProviderExample"></cookbook-code-viewer>
+
+  <p>
+    It can also be provided (or overridden if needed) in the component that uses
+    <code>kirby-calendar</code> like this:
+  </p>
+
+  <cookbook-code-viewer [ts]="componentProviderExample"></cookbook-code-viewer>
+
+  <h4>Adding additional existing locales:</h4>
+
+  <p>
+    To provide additional locales than the already built-in (Danish & English), simply import them
+    from <code>date-fns/locale</code> and pass it to <code>kirby-calendar</code> in an object.
+  </p>
+
+  <cookbook-code-viewer [ts]="additionalLocalesExampleTS"></cookbook-code-viewer>
+  <cookbook-code-viewer [html]="additionalLocalesExampleHTML"></cookbook-code-viewer>
+
+  <h4>Adding custom locales:</h4>
```

Comment:
is  colon (`:`) commonly used in the cookbook?

Reply:
I checked and it depends on which page on the documentation you're looking at 😅 I have removed them from the calendar documentation :)